Controller/CRUDController.php

```diff
@@ -456,6 +473,8 @@ public function editAction($id = null, Request $request = null)
                 $templateKey = 'preview';
                 $this->admin->getShow();
             }
+        } else {
+            $session->set($versionKey, $this->admin->getVersion($object));
```

Comment:
@rande The problem is that at this moment, the object version number will be hydrated from the database. We need to remember the version number when loading the form for the first time.

Let me try to add the version number in the form, which will be more complex to achieve but maybe more decoupled ?